### DISCUSSION TITLE
Can you make a private project for me?

### DISCUSSION THREAD

**Discussion by gamerquicksilver2099** [community/community#20971] [Other Feature Feedback, Questions, & Ideas]
N

**Comment by HD111995** [comment]
Here how to do it @gamerquicksilver2099 
https://www.youtube.com/watch?v=stXwoiBr6HI

**Comment by jfullstackdev** [comment]
here is the official documentation 

https://docs.github.com/en/get-started/quickstart/create-a-repo
